hub/powertoys/peek.md

```diff
@@ -1,15 +1,14 @@
 ---
 title: PowerToys Peek utility for Windows
 description: A system-wide utility for Windows to quickly preview file content.
-ms.date: 11/19/2024
-ms.topic: concept-article
```

Comment:
Please keep this new ms.topic type. It's one of our content patterns supported by some new tooling.